src/model/immutable/ContentState.js

```diff
@@ -56,6 +54,18 @@ class ContentState extends ContentStateRecord {
     return block;
   }
 
+  // return all blocks
```

Comment:
This comment looks out of date. Looking at the implementation of `getFirstLevelBlocks` and `getBlockChildren` it looks like this method really does return the first level blocks.